.github/CODEOWNERS

```diff
@@ -428,6 +428,13 @@
 
 
 # Infrastructure.
+/jobs/__init__.py @brianrodri
+/jobs/base_model_validator.py @brianrodri
+/jobs/base_model_validator_test.py @brianrodri
+/jobs/base_model_validator_errors.py @brianrodri
+/jobs/base_model_validator_errors_test.py @brianrodri
```

Comment:
`/jobs/base_model_*.py @brianrodri` [So that anything related to base_model_validator is assigned to @brianrodri ]